biogrid/libs/biogrid-simulator/src/lib/biobattery/biobattery-class.ts

```diff
@@ -1,18 +1,45 @@
-import { Battery, Energy, Validatable, validate } from "@biogrid/grid-simulator";
+import {
+  Battery,
+  Distance,
+  Energy,
+  ItemPosition,
+  Validatable,
+  validate,
+} from '@biogrid/grid-simulator';
 
 export class BioBattery implements Battery {
   private currentBatteryPower: Energy;
   private readonly maxCapacity: Energy;
+  private readonly position: ItemPosition;
```

Comment:
a bit of a higher level comment, but here we have position and in other classes we use lat and long.
It would be great to have a documentation of how these two are correlated (does position also represent a lat long?), or maybe instead of "position" we rename to "relativePosition" or something like that?

Reply:
Ok sure, changed to relativePosition and added a comment